packages/x-components/src/x-modules/experience-controls/components/experience-controls.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import Vue from 'vue';
+  import { Component } from 'vue-property-decorator';
+  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { XEvent, XEventsTypes } from '../../../wiring/events.types';
+
+  /**
+   *This component subscribes to changes in the ExperienceControls module to fire the events that propagate the configuration.
+   *
+   * @param events - The list of the events on the module.
+   */
+  import { xComponentMixin } from '../../../components/x-component.mixin';
+  import { experienceControlsXModule } from '../x-module';
```

Comment:
These imports shouldn't be here, they should at the top of the `scropt` with the rest.